tools/package_tools/configs/windows_ort.yaml

```diff
@@ -0,0 +1,15 @@
+global_config:
+  bdist_tags:
+    python_tag: "py37"
+  cmake_flags: ['-G "Visual Studio 17 2022" -A x64 -T v143']
+  cmake_envs:
```

Comment:
On my machine, I should add OpenCV_DIR and spdlog_DIR when building the windows_ort.yaml, or it can't find these library.